osu.Game/Overlays/Profile/Sections/Historical/UserHistoryGraph.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using static osu.Game.Users.User;
+
+namespace osu.Game.Overlays.Profile.Sections.Historical
+{
+    public class UserHistoryGraph : UserGraph<DateTime, long>
+    {
+        public UserHistoryCount[] Values
+        {
+            set => Data = value?.Select(v => new KeyValuePair<DateTime, long>(v.Date, v.Count)).ToArray();
+        }
+
+        /// <summary>
+        /// Text describing the value being plotted on the graph, which will be displayed as a prefix to the value in the <see cref="HistoryGraphTooltip"/>
+        /// </summary>
+        public string TooltipCounterName { get; set; } = @"Plays";
```

Comment:
I guess at this point it's better to make it just a ctor parameter, but I'm not sure